README.rst

```diff
@@ -22,3 +22,4 @@ Usage::
     # htmlmin.reduce_boolean_attributes = False
     # htmlmin.remove_optional_attribute_quotes = True
     # htmlmin.keep_pre = False
+    # htmlmin.gzip_if_accepted = True
```

Comment:
There is a comment above that says: 

> the settings here are default value of htmlmin.minify methods,
> provided by htmlmin version 0.1.5

So maybe separate a bit that comment from the others above.